### PR TITLE
Tagging Tool Added

### PR DESCRIPTION
**Pull request by Yer1k**
In this PR, the tagging tool is uploaded, and the READMEs are updated.

### CODE REVIEW

File: 10_code/tools/tagging_tool.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            st.session_state.current_image_index = filenames.index(
+                st.session_state.selected_filename
+            )
+            st.session_state.last_image_reached = False
+            st.experimental_rerun()
+
+        display_current_image(df_yes)
+    else:
+        st.error(
+            "😢 Oh no! We couldn't find data for the specified state. Double-check the name and try again?"
```
Comment (Yer1k) [inline 10_code/tools/tagging_tool.py:127]: Haha, I hope we do not run into this.

File: 10_code/tools/tagging_tool.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import streamlit as st
+import os
+import pandas as pd
+
```
Comment (Yer1k) [inline 10_code/tools/tagging_tool.py:4]: Do you mind me trying to add keyboard shortcuts?

**Review by 5ukhy21 — APPROVED** [review]
Looks good to me!

File: 10_code/tools/tagging_tool.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+new_title = '<p style="font-size: 42px;">🌟 Welcome to the WWTP and Solar Panel Tagging Tool 🌟</p>'
+st.markdown(new_title, unsafe_allow_html=True)
+
+# User input for state name
+state_name = st.text_input("👉 Enter the state name you want to tag:", value="").strip()
+
+
+# Function to display the current image and handle user responses
+def display_current_image(df_yes):
+    col1, col2 = st.columns([3, 2])  # Adjust column width ratios as needed
```
Comment (Yer1k) [inline 10_code/tools/tagging_tool.py:19]: We may want to add docstrings

**Review by Yer1k — COMMENTED** [review]
Awesome work

**Review by poojakabber7 — APPROVED** [review]

File: 10_code/tools/tagging_tool.py (hunk 2 above)
Comment (YZhu0225) [reply 10_code/tools/tagging_tool.py:4]: sure, go ahead.

File: 10_code/tools/tagging_tool.py (hunk 3 above)
Comment (YZhu0225) [reply 10_code/tools/tagging_tool.py:19]: Good catch, will add later.

**Review by YZhu0225 — APPROVED** [review]

**Pull request merged**